WeatherApp_Obj-C/WeatherApp_Obj-C/model/WeatherModel/City/City.m

```diff
@@ -0,0 +1,29 @@
+//
+//  City.m
+//  WeatherApp_Obj-C
+//
+//  Created by Muradasil Birhan on 24/02/2020.
+//  Copyright © 2020 RodeApps. All rights reserved.
+//
+
+#import "City.h"
+#import "Coord.h"
+
+@implementation City
+
+- (instancetype)initWithJSON:(NSDictionary *)dict {
+    self = [super init];
+    if (self) {
+        self.sunset = [[dict valueForKey:@"sunset"] longValue];
```

Comment:
Rather than accessing the key with KVC ("valueForKey", which only takes strings as keys), use the dedicated NSDictionary method objectForKey